app/Console/Commands/UpdateOrchids.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\Profile;
+use App\ProfileData;
+
+
+class UpdateOrchids extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'orchid:update';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Loop through the profiles with orcid managed publications to sync the info';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $inc = 0;
+        $profiles_to_update = Profile::with('data')->get();
```

Comment:
Great first start. For this query, I think you might be able to pre-filter via a JSON selector so you don't have to iterate over each profile and manually check. Will likely be more time efficient, but will more likely be more memory efficient. 

@wunc can confirm or provide more specific feedback.